docs/setup-cluster/k8s/install-on-kubernetes.rst

```diff
@@ -425,17 +459,20 @@ Determined <configure-determined-kubernetes-version>` to install on Kubernetes.
  Upgrade Determined
 ********************
 
-To upgrade Determined or to change a configuration setting, first make the appropriate changes in
-``values.yaml`` and ``Chart.yaml``, and then run:
+To upgrade Determined or to change a configuration setting, update the helm repo, make the
+appropriate changes in ``values.yaml``, and then run:
 
 .. code::
 
-   helm upgrade <name for your deployment> --wait determined-helm-chart
+   helm repo update
```

Comment:
The sentence above says "update the helm repo ... and then run" , so either that or this `helm repo update` is redundant.